.gitignore

```diff
@@ -378,9 +378,9 @@ FodyWeavers.xsd
 
 # VS Code files for those working on multiple tools
 .vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
+.vscode/settings.json
```

Comment:
Meant for these to be ignored, accidentally left the `!` in

Reply:
Shouldn't the line above cover this ignore then?